l10n/en/products/vpn/more/ip-address.ftl

```diff
@@ -48,4 +48,5 @@ vpn-ip-address-how-do-you = How do you hide it?
 # Variables
 #   $vpn (url)- https://www.mozilla.org/products/vpn/more/what-is-a-vpn/
 #   $mozvpn (url) - https://www.mozilla.org/products/vpn/
-vpn-ip-address-a-vpn-is = A VPN is a way to hide your IP address. <a href="{ $vpn }">When you use a VPN</a>, your external IP address will be coming from the VPN server’s external IP, rather than your location’s external IP address. So if your connecting VPN server is located in California, your external IP will look like it’s connected from California, no matter where you actually are. Plus, your online activity is sent over an encrypted, secure connection to your VPN server, giving you additional security and privacy. <a href="{ $mozvpn }">{ -brand-name-mozilla-vpn }</a> is one way to hide your IP address. We don’t keep activity logs or partner with third-party analytics platforms. We offer full-device protection for up to five devices with servers in 30+ countries, you can connect to anywhere, from anywhere.
+#   $countries (number) - number of countries where Mozilla VPN has servers, e.g. "30+"
```

Comment:
Suggest leaving the `+` in the fluent string here (see above) so the variable is strictly a number.